networks/mainnet.json

```diff
@@ -267,6 +277,25 @@
             "decimals": 18,
             "address": "0xE41d2489571d322189246DaFA5ebDe1F4699F498"
         },
+        "cUSDC": {
```

Comment:
were these cTokens mean to be added back to the "Tokens" object?

Reply:
No, they weren't this is in `cTokens`